src/Shared/EmptyState.js

```diff
@@ -72,47 +86,70 @@ export default class EmptyListWrapper extends Component {
             </View>
           </View>
         )}
-      </>
+      </View>
     )
   }
 }
 
 function ImageHolder(props) {
-  let { imageSource, emptyStateImageStatus } = props
+  let {
+    imageSource,
+    emptyStateImageStatus,
+    textTitleDisplay,
+    imageWidth,
+    imageHeight,
+    imageRounding,
+  } = props
   let realImageSource = !imageSource
     ? require('./sqr-empty-state.png')
     : imageSource
+
+  const imageWrapperSize = { width: imageWidth, height: imageHeight }
+  const imageBorderRadius = {
+    borderRadius:
+      imageHeight < imageWidth
+        ? (imageHeight / 2) * (imageRounding / 100)
```

Comment:
what's all this for? I just don't see anything in the card

Reply:
Originally the card said to default to max rounding, it must have gotten removed when we decided we weren't going to add rounding controls here though. I can remove this now and just leave the image as a square if that's how we want to handle it.